pkg/check/act/act.go

```diff
@@ -0,0 +1,179 @@
+package act
+
+import (
+	"bytes"
+	"context"
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/ethersphere/bee/pkg/swarm"
+	"github.com/ethersphere/beekeeper/pkg/bee"
+	"github.com/ethersphere/beekeeper/pkg/bee/api"
+	"github.com/ethersphere/beekeeper/pkg/beekeeper"
+	"github.com/ethersphere/beekeeper/pkg/logging"
+	"github.com/ethersphere/beekeeper/pkg/orchestration"
+	"github.com/ethersphere/beekeeper/pkg/random"
+)
+
+// Options represents check options
+type Options struct {
```

Comment:
options could be configurable in config.yaml and local.yaml, and then mapped in pkg/config/check.go where "act" is defined.

Reply:
done

https://github.com/Solar-Punk-Ltd/beekeeper/blob/feat/act/config/config.yaml#L239
https://github.com/Solar-Punk-Ltd/beekeeper/blob/feat/act/config/local.yaml#L219
https://github.com/Solar-Punk-Ltd/beekeeper/blob/feat/act/pkg/config/check.go#L64